webapp/src/actions/index.ts

```diff
@@ -424,6 +424,10 @@ export function handleConnectChange(store) {
             return;
         }
 
+        if (msg.event === 'custom_jira_connect') {
+            store.dispatch(sendEphemeralPost('You have successfully connected your Jira account.'));
```

Comment:
```suggestion
            store.dispatch(sendEphemeralPost('You have successfully connected your Jira account. Type in /jira to get started. '));
```

Reply:
Sorry - Github gave me some issues today.... the comment didn't post.  Added it now. small change to text of what bot says.